### PR TITLE
Add close button to sticky header in ProductHeroV2

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add close button to sticky header in ProductHeroV2

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/nv8d5VeBPARA7ezh3Od3/fe4b636b-951c-4b4d-839a-f29f353c37a6.mov">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/nv8d5VeBPARA7ezh3Od3/fe4b636b-951c-4b4d-839a-f29f353c37a6.mov">Screen Recording 2024-10-24 at 09.38.41.mov</video>

## Justify why they are needed
Allow user to close down the price calculator in mobile
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4893** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4893?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/ProductHeroV2/ProductHeroV2.css.ts [hunk 1]
```diff
@@ -75,3 +75,11 @@ export const subTypeBadge = style({
     },
   }),
 })
+
+export const mobileCloseButton = style([
+  sprinkles({
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/ProductHeroV2/ProductHeroV2.css.ts:80]: Neat! Should we prefer this to absolute positioning in the future?

**Review by alebedev — APPROVED** [review]

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/features/priceCalculator/ProductHeroV2/ProductHeroV2.css.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/features/priceCalculator/ProductHeroV2/ProductHeroV2.css.ts:80]: When applicable I'm a big fan of margin auto :) Like in flex containers when you want to offset a single child

**Comment by gustaveen** [comment]
### Merge activity

* **Oct 24, 4:52 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4893).

**Pull request merged**